### DISCUSSION TITLE
Trino on yarn

### DISCUSSION THREAD

**Discussion by ml-jin** [trinodb/trino#6794] [General]
Hi every gurus,

Are there any road maps to Trino-on-yarn? If not, why?  I found some old docs said that we can presto-on-yarn plugins (include slider), but  it is not maintained anymore. 

Thx!

**Comment by kokosing** [comment]
There are no such plans. We do not recommend sharing node with other components on it.

The integration with slider was complex and hard to maintain. Also it was not used frequently, so we decided to abandon this.

Can you please share more about your use case and why you think you need Trino-on-yarn?

**Reply by ml-jin** [reply]
Hey,

Thx for reply. I asked mainly because my company's startegy and policy with the big data component. we basically want everything, every component using yarn. your words make sense to me.  In my mind, purely using presto cluster makes it more stable and performed well.

Thx!

**Comment by findepi** [comment]
For reference, here is the project @kokosing had in mind: https://github.com/trinodb/trino-yarn

But to @kokosing 's point. We strongly advise not to use yarn to run Trino nodes, this is not supported. If you choose this path, you're on your own.
See https://trino.io/resources.html#tools for supported tools (ups, presto-admin, hasn't been updated yet for Trino)
or join [our slack](https://trino.io/slack.html) where you can ask the community how they deploy Trino clusters 
(people who are active on slack, not necessarily watch github discussions)

**Reply by ml-jin** [reply]
Thx.  I will do research on it :-)
